contribute/en/Style-and-Tone.md

```diff
@@ -109,3 +109,5 @@ Kbit/s; Mbit/s; KByte/s; MByte/s.
 
 Values followed by units such as `ms`, `s`, `m`, `km`, `GHz` etc. should be separated by a space; e.g. "20 ms" and "1 GHz" not "20ms" and "1GHz".
 
+Note that we do not distinguish between 1024-based and 1000-based values for these.
```

Comment:
Only applies to the bits and bytes.  Should be part of the same paragraph.  Does not apply to time.

Reply:
BTW I take it we're still OK to merge this direct to release? I was just wary of the recent issues with Weblate.